docker/test/fuzzer/run-fuzzer.sh

```diff
@@ -368,6 +368,7 @@ if [ -f core.zst ]; then
 fi
 
 rg --text -F '<Fatal>' server.log > fatal.log ||:
+dmesg -T > dmesg.log ||:
```

Comment:
Consider adding `dmesg --clear` at the beginning of the script to avoid overlaps with other containers.